src/developer/web-api/analytics.md

```diff
@@ -2128,6 +2129,20 @@ Data elements and tracked entity attributes are considered *supported types* if
 - `BOOLEAN`
 - `TRUE_ONLY`
 
+### Tracked Entities analytics dimensions
+
+#### Tracked Entities query analytics dimensions { #webapi_teis_query_analytics_dimensions }
+
+The `/analytics/teis/query/dimensions?trackedEntityType=TET` resource accepts a mandatory id of a tracked entity type `TET` and returns the following data dimensions:
+
+for each program `P` such that a Tracked Entity Instance of type `TET` can register in program `P` :
```

Comment:
Should we use all lowercase as we do for program, data element, etc.?
`Tracked Entity Instance` -> `tracked entity instance`